cms/envs/common.py

```diff
@@ -1079,7 +1079,7 @@
 LANGUAGE_CODE = 'en'  # http://www.i18nguy.com/unicode/language-identifiers.html
 LANGUAGES_BIDI = lms.envs.common.LANGUAGES_BIDI
 
-LANGUAGE_COOKIE = lms.envs.common.LANGUAGE_COOKIE
+LANGUAGE_COOKIE_NAME = lms.envs.common.LANGUAGE_COOKIE_NAME
```

Comment:
Django needs the settings name to be this. So that's why changed this everywhere in the codebase

> The name of the cookie used is set by the LANGUAGE_COOKIE_NAME setting. (The default name is django_language.)

https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#how-django-discovers-language-preference